openflow13/nxext_test.go

```diff
@@ -0,0 +1,215 @@
+package openflow13
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"testing"
+)
+
+func TestNXActionResubmit(t *testing.T) {
+	tableID := uint8(10)
+	portID := uint16(10)
+
+	action1 := NewNXActionResubmit(portID)
+	data1, err := action1.MarshalBinary()
+	if err != nil {
+		t.Errorf("Failed to invoke NXAST_RESUBMIT_TABLE.MarshalBinary: %v", err)
```

Comment:
should be a `Fatalf` IMO. If the call fails then `data1` is not correct and there is no point in calling `UnmarshalBinary` on it after.

This probably applies to other locations as well.

Reply:
Sure，I would update.

Reply:
I do think there are more places where it should be `Fatalf` and not `Errorf`. Basically you have to ask yourself: does it make sense to proceed with the test if this fails, or will everything else just fail automatically because of this one failure.